app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,205 @@
+/* Global Styles */
+
+html,
+body {
+    height: 100%;
+}
+
+body {
+    padding-top: 42px; /* Required padding for .navbar-fixed-top. Remove if using .navbar-static-top. Change if height of navigation changes. */
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: 400;
+}
+
+.navbar {
+  font-weight: 400;
+}
+
+.page-header {
+  border-bottom: none;
+  color: white;
+  font-weight: 500;
+  float: right;
+}
+
+.featured {
+  color: black;
+  float: none;
+}
+.main_image {
+  background: url('http://f.fwallpapers.com/images/dew-drops-carnivorous-plant.jpg') no-repeat center center;
```

Comment:
Prefer double-quoted strings<br>URL `http://f.fwallpapers.com/images/dew-drops-carnivorous-plant.jpg` should not contain protocol or domain